frontend/packages/dev-console/src/components/topology/shapes/WorkloadNode.tsx

```diff
@@ -62,6 +69,35 @@ const WorkloadNode: React.FC<NodeProps<WorkloadData>> = ({
             </g>
           </Decorator>
         ),
+        build && (
+          <Link
+            to={`/k8s/ns/${build.metadata.namespace}/buildconfigs/${
+              build.metadata.ownerReferences[0].name
+            }`}
+          >
+            <Decorator
+              key="build"
+              x={-radius + decoratorRadius * 0.7}
+              y={radius - decoratorRadius * 0.7}
+              radius={decoratorRadius}
+              title={`${build.metadata.name} ${build.status && build.status.phase}`}
+            >
+              <g transform={`translate(-${decoratorRadius / 2}, -${decoratorRadius / 2})`}>
+                {build.status.phase === 'Complete' ? (
+                  <GreenOutlinedCheckCircleIcon />
+                ) : (
+                  <foreignObject width={decoratorRadius} height={decoratorRadius}>
```

Comment:
Icon is clipped.
![image](https://user-images.githubusercontent.com/14068621/62059417-4129bd80-b1f1-11e9-86f5-901b2f66d5f8.png)

See other icons which are based on the `decoratorRadius` and set the `font-size` accordingly.

Reply:
@christianvogt I'm using `foreignobject` and increasing it's height seems to fix the clipped icon.
![Screenshot from 2019-07-30 10-20-04](https://user-images.githubusercontent.com/9278015/62101610-a0e89d00-b2b3-11e9-809b-c1bedfe0feac.png)

Reply:
Look at the other icons. They are all using a font size of `decoratorRadius`, please follow the pattern of the other icons. Adding `1` is not a suitable solution as everything else is dynamically sized.

Reply:
fixed https://github.com/openshift/console/pull/2193/commits/53d6ce5f316ad3948709ce9f12d589f4aaa253c2